tests/asserts/keywords.py

```diff
@@ -370,6 +370,26 @@ def process_strats(src, paramData):
             for [k, v] in flattened_dict.items():
                 process_keyvalue(k, v, {})
 
+# process connections to identify vertical doors:
+vertical_door_nodes = set()
+connections = {
+    "inter": {},
+    "intra": {},
+    "subarea": {}
+}
+connectionPath = os.path.join(".","connection")
+for root, dirs, files in os.walk(os.path.join(".", "connection")):
+    for filename in files:
+        if not filename.endswith(".json"):
+            continue
+        with open(os.path.join(root, filename), "r", encoding="utf-8") as connectionFile:
+            connections_json = json.load(connectionFile)
+            for connection in connections_json["connections"]:
+                for i, node in enumerate(connection["nodes"]):
+                    if node["position"] in ["top", "bottom"]:
```

Comment:
`connectionType` in tears 😢

Reply:
True, `connectionType` is there, and with that we could easily exclude the elevators, which might be better.

Reply:
Ok, I switched it over to use `connectionType` now and excluded the elevators since the data isn't meaningful for them.

I found one case where the `connectionType` was wrong, so this also fixes that.